x-pack/legacy/plugins/apm/server/routes/ui_filters.ts

```diff
@@ -136,6 +139,31 @@ export function initUIFiltersApi(core: InternalCoreSetup) {
         transactionName?: string;
       };
       return getTransactionGroupsProjection({
+        setup,
+        options: {
+          type: 'top_transactions',
+          transactionType,
+          serviceName
+        },
+        transactionName
```

Comment:
Perhaps `transactionName` should be inside `options`? Or the items in `options` should be elevated to the parent level?